### ISSUE TITLE
Sometimes loading a save from death dialog breaks

### ISSUE THREAD

**Issue by daerogami** [calref/cboe#140]
---
This is an item logged from the To-Do and was originally reported by ADoS. If you are familiar with or can recreate this bug, please provide details here.

**Comment by NQNStudios** [comment]
I think there might not be enough information to go off of here? Although with the new "Kill party" debug action I could just try doing it a bunch.

**Comment by CelticMinstrel** [comment]
I think I recall something you recently fixed that could be related, maybe…?

**Comment by NQNStudios** [comment]
Yeah, first it was the scrollbar error, then after we fixed that there was a new error. I just don't know what this issue means by "breaks"--it sounds more like crashing than like the scrollbar error.

I don't know if we fixed anything in this code path before the scrollbar error. Maybe a blame will show it.

**Comment by NQNStudios** [comment]
11 years ago you updated the relevant code path in this commit: https://github.com/calref/cboe/blame/d95703ecf53a1ad51a82256bb61b4604207a3894/src/game/boe.actions.cpp

>Fix a plethora of bugs and crashes when loading new savegames; loading now works, though with a few glitches still

**Comment by NQNStudios** [comment]
I had to do a lot of this while playtesting VoDT and ASR, I think it's fixed.
